### PR TITLE
Demo updates

### PR DESCRIPTION
**Pull request by rhlsthrm**

### CODE REVIEW

File: packages/contracts/test/transactionManager.spec.ts [hunk 1]
```diff
@@ -705,7 +705,7 @@ describe("TransactionManager", function () {
       ).to.be.revertedWith("prepare: INSUFFICIENT_LIQUIDITY");
     });
 
-    it("happy case: prepare by Bob for ERC20", async () => {
+    it.only("happy case: prepare by Bob for ERC20", async () => {
```
Comment (LayneHaber) [inline packages/contracts/test/transactionManager.spec.ts:708]: Remove

File: packages/sdk/src/utils/listener.ts [hunk 2]
```diff
@@ -4,40 +4,42 @@ import {
...
+import { TransactionManager } from "@connext/nxtp-contracts/typechain";
+import { providers } from "ethers";
 import { Evt } from "evt";
 
-import { NxtpSdkEvent, NxtpSdkEvents, TransactionCompletedEvent, NxtpSdkEventPayloads } from "../sdk";
-
 import { getTransactionManagerContract } from "./contract";
-// Define event types
+
+export const TransactionManagerEvents = {
```
Comment (LayneHaber) [inline packages/sdk/src/utils/listener.ts:13]: Do the sender/receiver ones need to be here?

**Review by LayneHaber — CHANGES_REQUESTED** [review]
Just remove the .only

File: packages/sdk/src/utils/listener.ts (hunk 2 above)
Comment (rhlsthrm) [reply packages/sdk/src/utils/listener.ts:13]: The way I'm thinking about this is the TransactionManagerListener class is directly tied to the contract so it mirrors the contract events exactly. At the higher SDK level, we consume these events and translate them into the SDK specific events which are sender/receiver.

**Pull request merged**